sound/soc/sof/intel/hda-pcm.c

```diff
@@ -118,7 +118,7 @@ int hda_dsp_pcm_hw_params(struct snd_sof_dev *sdev,
 
 	/* set host_period_bytes to 0 if no IPC position */
 	if (hda && hda->no_ipc_position)
-		ipc_params->host_period_bytes = 0;
+		ipc_params->period_irq = 0;
```

Comment:
In this series, period_irq is never set to anything different from zero, so why do we need this parameter. Something's missing I guess

Reply:
Its not used here but FW uses it:

'
/* NO_IRQ mode if period_irq == 0 */
	if (dev->params.period_irq != 0) {
'
ALthough you won't find it in current master, I will submit a PR to master soon.